packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -276,18 +288,26 @@ const ApiTokenCreateView = () => {
                   }
                   primaryAction={
                     canEditInputs && (
-                      <Button
-                        disabled={isSubmitting}
-                        loading={isSubmitting}
-                        startIcon={<Check />}
-                        type="submit"
-                        size="S"
-                      >
-                        {formatMessage({
-                          id: 'global.save',
-                          defaultMessage: 'Save',
-                        })}
-                      </Button>
+                      <Flex justifyContent="center">
```

Comment:
Why do you need a flex here? Could you use `Stack` instead, to handle the spacing between both buttons?

Reply:
done